.ci/generate_redirects.jl

```diff
@@ -90,10 +90,12 @@ function create_redirect_page(; name, path)
 end
 
 function main()
-    mkpath(package_path())
-    packages_info = get_packages_info()
-    for (; name, path) in packages_info
-        create_redirect_page(; name, path)
+    cd(joinpath(@__DIR__, "..")) do
+        mkpath(package_path())
+        packages_info = get_packages_info()
+        for (; name, path) in packages_info
+            create_redirect_page(; name, path)
+        end
```

Comment:
I think no change is needed here, since the cwd hasn't changed. That being said this is maybe a bit more robust since it doesn't rely on the cwd